plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -159,6 +161,26 @@ const fetchPluginData = async ({createNode, reporter}) => {
         page = pluginsContainer.page + 1;
     } while (!page || pluginsContainer.page < pluginsContainer.pages);
     await Promise.all(promises);
+    const updateUrl = 'https://updates.jenkins.io/current/update-center.actual.json';
+    const updateData = await requestGET({url: updateUrl, reporter});
+    const suspendedPromises = [];
+    for (const deprecation of Object.keys(updateData.deprecations)) {
+        if (names.indexOf(deprecation) == -1) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes
I recommend `names.includes(deprecation)` instead of indexof. Also I think performance wise, a set is faster than a straight up array, but i doubt it matters for this.